gdpr/impl.go

```diff
@@ -98,41 +98,42 @@ func (p *permissionsImpl) allowSync(ctx context.Context, vendorID uint16, consen
 	return false, nil
 }
 
-func (p *permissionsImpl) allowPI(ctx context.Context, vendorID uint16, consent string) (bool, bool, error) {
+func (p *permissionsImpl) allowPI(ctx context.Context, vendorID uint16, consent string) (bool, bool, bool, error) {
```

Comment:
To confirm, this is for TCF1 which doesn't have the same purpose breakout, so it's all or nothing here.

Reply:
Correct

Reply:
> I didn't see removal of the Purpose 4 check.

Purpose 4 never made it in to the GDPR code. So it doesn't exist to remove.

Reply:
Ah. Gotcha. It was never part of the original spec. It was added and subsequently removed before we pushed out an update.